config/Hipcheck.kdl

```diff
@@ -20,16 +20,17 @@ analyze {
     category "practices" {
         analysis "mitre/activity" policy="(lte $ 52)" weight=3
         analysis "mitre/binary" {
-			binary-file "./config/Binary.toml"
-			binary-file-threshold "0"
+			binary_file "./config/Binary.toml"
+			binary_file_threshold 0
 		}
         analysis "mitre/fuzz" policy="(eq #t $)"
         analysis "mitre/review" policy="(lte $ 0.05)"
     }
 
     category "attacks" {
-        analysis "mitre/typo" policy="(eq 0 (count $))" {
-            typo-file "./config/Typos.toml"
+        analysis "mitre/typo" {
+            typo_file_path "./config/Typos.toml"
+            count_threshold 0
```

Comment:
`typo` plugin should be changed to take kebab-case config fields, not the other way around.